docs/tutorials/index.md

```diff
@@ -1,6 +1,6 @@
 # Tutorials
 
-These tutorials introduce fundamental concepts in deep learning and their realizations in _MXNet_. Under the _basics_ section, you'll find tutorials covering manipulating arrays, building networks, loading and preprocessing data, etc. Training and Inference section will introduce how to implement Linear Regression in MXNet, How to train a Hand digit classifier using MLP and CNNs and run inference on the trained model, run inference using pre-trained models and lastly how to efficiently train a large scale image classifier.
+These tutorials introduce the fundamental concepts in deep learning and how to implement them in _MXNet_. The _Basics_ section contains tutorials on manipulating arrays, building networks, loading/preprocessing data, etc. The _Training and Inference_ section talks about implementing Linear Regression, training a Handwritten digit classifier using MLP and CNN, running inferences using a pre-trained model, and lastly, efficiently training a large scale image classifier.
```

Comment:
My previous comment is probably not justified after I re-read your PR description.

I agree with 1, 2, 4, 5.